### ISSUE TITLE
Error trying to format ngClass attribute with optional chaining followed by property access via variable

### ISSUE THREAD

**Issue by TomTomB** [tailwindlabs/prettier-plugin-tailwindcss#91]
**What version of `prettier-plugin-tailwindcss` are you using?**

For example: v0.1.13

**What version of Tailwind CSS are you using?**

For example: v3.1.8

**What version of Node.js are you using?**

For example: v16.16.0

**What package manager are you using?**

yarn 2 (pnp disabled)

**What operating system are you using?**

Windows 11

**Reproduction URL**

https://github.com/TomTomB/tailwind-ng-prettier-issue

**Describe your issue**

The repo is a bare nrwl nx setup.
After clone, the following commands needs to get executed:

- npm i
-  npm run prettier .\apps\issue-app\src\app\app.component.html

This will result in the following error.

```
apps\issue-app\src\app\app.component.html[error] apps\issue-app\src\app\app.component.html: Error: Unexpected node undefined
```

**Comment by thecrypticace** [comment]
Hey thanks for the report — this was a confusing bug for sure. I've come up with a fixed and merged it in. We're likely gonna do some extra bits of work before we tag our next release though. I'm hoping by the end of this week if we can swing it but not 100% certain.

**Comment by rosslavery** [comment]
Any chance you can cut a release to include the fixes from #93 ? We're being bitten by this exact issue :(
